db/migrate/20171119094913_add_mtu_to_subnet.rb

```diff
@@ -0,0 +1,5 @@
+class AddMtuToSubnet < ActiveRecord::Migration
+  def change
+    add_column :subnets, :mtu, :integer
```

Comment:
This does not store `4294967295` on mysql, as passing no limit to the migration makes rails create a `integer` column on mysql. The maximum value is `2147483647`. By setting something between `limit: 5` and `limit: 8` causes rails to create a bigint column with a maximum value of `9223372036854775807`.

Reply:
Done.